Makes improvements to new computer setup script.

amcheshire opened the pull request:
I made this changes as part of setting up my new computer.

## Checklist
- [x] I've reviewed the entire diff
- [x] all changes are essential to what we're trying to achieve
- [x] no style issues
- [x] ~documentation added/updated as necessary (using [tags](https://docs.google.com/document/d/1OFrkFQHIA7ucvHyE5EqAnkPuWwGwQ04CYFD0VpF7Xhs/edit) on Drive should make this easier)~ none

Commit:
Makes improvements to new computer setup script:

- Removes a git configuration (conflictstyle diff3) since people have different preferences for this.
- Adds homebrew's installation of postgresql@9.6 to PATH. This does not happen upon installation, so we have to do it ourselves.
- Removes code to install latest version of Ruby and Node through asdf. We are usually not on the latest version, so this is not very helpful. I also think that not having any versions to begin with will help new hires learn how to use asdf.

amcheshire commented on mac:
```diff
@@ -115,7 +115,6 @@ tap "caskroom/cask"
 brew "universal-ctags", args: ["HEAD"]
 brew "git"
 # git customization
-git config --global merge.conflictstyle diff3
```
This was causing errors, but I didn't try to figure out why because I don't think we should be doing this anyways. People have different preferences for this, so I think they should change it if they want.

cdeevfrr replied:
*sad face*

tmr08c reviewed (approved):
Seems reasonable.

amcheshire commented on mac:
```diff
@@ -155,6 +154,11 @@ if brew list | grep --silent "qt@5.5"; then
 
+if [ "$(which psql)" == "" ] ; then
+  fancy_echo "'psql' not found in PATH, attempting to add it..."
+  append_to_bash_profile 'export PATH="/usr/local/opt/postgresql@9.6/bin:$PATH"'
```
This needs to be added to the PATH, but homebrew doesn't do this on installation. I've always had to do it manually on every laptop setup, so I instead automated it here.

I'm not sure if the installation path is always the same on every machine, so I'm not sure if this will work for everyone.

cdeevfrr replied:
Will this interact with other path settings, like RVM which says it has to be last?

amcheshire replied:
It shouldn't. If something needs to be last, it just needs to be put in the right place on the `bash_profile`, which is true anyways and the responsibility of the installer.

amcheshire commented on mac:
```diff
@@ -187,17 +191,6 @@ install_asdf_language() {
-
-fancy_echo "Installing latest Node..."
-bash "$HOME/.asdf/plugins/nodejs/bin/import-release-team-keyring"
-install_asdf_language "nodejs"
```
This code installed the latest version of Ruby and Node. We are usually not on the latest version, so I didn't think this was very helpful. I also think that not having any versions to begin with will give new hires a good opportunity to learn how to use asdf early on.

cdeevfrr replied:
However, being able to run node or ruby from the command line might be helpful. It also might help with setup instructions, for example, can't ruby install the necessary versions of itself using bundler? Or does everything always have a version manager independent of the code?

amcheshire replied:
I think that if someone wants to be able to do these things outside of an app repo, they should be responsible for picking the right version. I don't think we should assume it's always whatever the latest version happens to be.

Commit:
Changes how we add asdf to the PATH.

Rather than 'source'ing, we manually prepend the paths to PATH.

We do this because of how the '.bash_profile' is loaded when opening a VS Code terminal. For some reason, it runs it twice, in which case all of the other prepended paths will go in front of 'asdf', which casues problems.

amcheshire commented on mac:
```diff
@@ -155,11 +154,15 @@ if brew list | grep --silent "qt@5.5"; then
   git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.4.2
-  append_to_bash_profile "source $HOME/.asdf/asdf.sh" 1
-  append_to_bash_profile "source $HOME/.asdf/completions/asdf.bash" 1
+  append_to_bash_profile 'export PATH="$HOME/.asdf/shims:$HOME/.asdf/bin:$PATH"' 1
```
It's better to prepend, rather than `source` because of how `.bash_profile` is run when opening a VS Code terminal.

For some reason, it's run twice, so everything that prepends does so twice. This puts a lot of things in front of `asdf` (since `source`ing twice does nothing), which causes issues.

cdeevfrr reviewed (approved):
If these things worked, they seem good to me! 

A bit worried about not having a starting ruby or a starting node, but if you think it'll work I trust it.
